frontend/src/ui/components/Header/Header.tsx

```diff
@@ -26,6 +26,7 @@ export const Header: React.FC = () => {
         <Navbar.Collapse id="basic-navbar-nav">
           <Nav className="ml-auto">
             <Route path={ROUTES.Registration.HOME}>
+              {(FAQ_PAGE_LINK.length > 0) && <Nav.Link href={FAQ_PAGE_LINK}>FAQ</Nav.Link>}
```

Comment:
Since this is a change to the Header.tsx , once there's a real URL in `REACT_APP_FAQ_PAGE_LINK`, you'll need to update the snapshot used in Header.spec.tsx.

Reply:
NVM, is environment-driven, so tests should still pass.